requirements.txt

```diff
@@ -2,4 +2,5 @@ numpy>=1.13
 scipy>=0.19
 pandas>=0.19.2
 scikit-learn>=0.18
-cloudpickle
\ No newline at end of file
+cloudpickle
```

Comment:
if you add something here, you should also add it in the `install_requires` list in setup.py (that is the one which is used when pip installing ramp-workflow)

Reply:
Was done by @aboucaud.